From 3146feb6f49fc21555388fdab989cc36abfb4e5e Mon Sep 17 00:00:00 2001
From: Vignesh Viswanathan <quic_viswanat@quicinc.com>
Date: Fri, 29 Nov 2024 12:38:16 +0530
Subject: [PATCH] remoteproc: qcom_common: Add declarations for userpd
 start/stop

Add declarations for q6v5_start_user_pd and q6v5_stop_user_pd.

Change-Id: Icacbd63cfce840882feaf6c19f579cee6cb162d6
Signed-off-by: Vignesh Viswanathan <quic_viswanat@quicinc.com>
---
 drivers/remoteproc/qcom_common.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/remoteproc/qcom_common.h b/drivers/remoteproc/qcom_common.h
index 9f3fb11224aa..ec345aa479dd 100644
--- a/drivers/remoteproc/qcom_common.h
+++ b/drivers/remoteproc/qcom_common.h
@@ -52,6 +52,9 @@ void qcom_add_ssr_subdev(struct rproc *rproc, struct qcom_rproc_ssr *ssr,
 			 const char *ssr_name);
 void qcom_remove_ssr_subdev(struct rproc *rproc, struct qcom_rproc_ssr *ssr);
 
+int q6v5_start_user_pd(struct rproc *rproc);
+int q6v5_stop_user_pd(struct rproc *rproc);
+
 #if IS_ENABLED(CONFIG_QCOM_SYSMON)
 struct qcom_sysmon *qcom_add_sysmon_subdev(struct rproc *rproc,
 					   const char *name,
-- 
2.34.1

